ocaml/testsuite/tests/typing-layouts/annots-failing.ml

```diff
@@ -0,0 +1,52 @@
+(* TEST
```

Comment:
I didn't review this file; I'll review it once they pass

Reply:
Sorry -- I missed this question. No. We'll come back to these later. There is an internal ticket in v1.5.